tensorboard/plugins/pr_curve/summary.py

```diff
@@ -55,8 +55,8 @@ def op(
     num_thresholds: Number of thresholds, evenly distributed in `[0, 1]`, to
         compute PR metrics for. Should be `>= 2`. This value should be a 
         constant integer value, not a Tensor that stores an integer.
-    weight: Optional; A float or scalar float32 `Tensor`. Individual
-        counts are multiplied by this value.
+    weights: Optional float32 `Tensor`. Individual counts are multiplied by this
```

Comment:
While you're updating docs, please note the shape of this argument. It is especially unclear because the default value is `1.0`, but any useful user-supplied value will need to be rank-1.